tests/spread/core24/python-hello/classic/snap/snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: python-hello-classic
+version: "1.0"
+summary: simple python application
+description: build a python application using core24
+base: core24
+build-base: devel
+confinement: classic
+
+apps:
+  python-hello-classic:
+    command: bin/hello
+    environment:
+      # Note that since commit 9b6e58337e0 we need to explicitly prepend $SNAP/bin
```

Comment:
This bit here is for when Sergio finally lets me fix classic Python snaps

Reply:
I would actually change the command to `bin/python3 bin/hello`, but let's fix the issue at the core if you want :-), PATH in environment could lead to issues

Reply:
updated this to use `bin/python3`, updated the note
another motivation to fix this is that a lot of users "fix" this issue by updating the PATH, which I agree is dangerous